src/modules/tests/test-data/cat-d/test-requirements/test-requirements.cat-d.reducer.ts

```diff
@@ -0,0 +1,23 @@
+import * as testRequirementsActions from '../../cat-b/test-requirements/test-requirements.actions';
```

Comment:
actions from cat-b

Reply:
Have moved these into common as all the same